client/GameData/RealmSynergy.cs

```diff
@@ -81,6 +82,7 @@ static RealmSynergy()
                 new SynergyValue("XII", 112001, Value.FF12),
                 new SynergyValue("XIII", 113001, Value.FF13),
                 new SynergyValue("XIV", 114001, Value.FF14),
+                new SynergyValue("Tactics", 150001, Value.FFT),

```

Comment:
Yeah, it really is 150001. I assume they're reserving the next numbers for the next numbered Final Fantasies. So FFXV will be 115001, etc.

The thing I was mentioning was in regards to the `series` table in the DB. The normal `ffrki-user` account can't modify that table, so we can't add new series to that, which means we can't add new equipment to the DB (which the `ffrki-user` can't anyway, though) due to FK constraints, etc.